<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>0</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trang Chủ</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>281</width>
      <height>701</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
	background-color:rgb(255, 255, 127);
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>281</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	background-color:rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>img/unnamed.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnTrangChu">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font-size: 20px;
    border-radius: 10px;
	background-color: rgb(0, 85, 255);
	color:rgb(255, 255, 255);
	font-weight: bold;
}
QPushButton:hover{
	background-color: rgb(0, 170, 255);
	cursor: pointer;
}</string>
     </property>
     <property name="text">
      <string>Trang chủ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnQuanLyPhim">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font-size: 20px;
    border-radius: 10px;
	background-color: rgb(0, 85, 255);
	color:rgb(255, 255, 255);
	font-weight: bold;
}
QPushButton:hover{
	background-color: rgb(0, 170, 255);
	cursor: pointer;
}</string>
     </property>
     <property name="text">
      <string>Quản lý phim</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnQuanLyVe">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font-size: 20px;
    border-radius: 10px;
	background-color: rgb(0, 85, 255);
	color:rgb(255, 255, 255);
	font-weight: bold;
}
QPushButton:hover{
	background-color: rgb(0, 170, 255);
	cursor: pointer;
}</string>
     </property>
     <property name="text">
      <string>Quản lý vé</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnQuanlyDichVu">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font-size: 20px;
    border-radius: 10px;
	background-color: rgb(0, 85, 255);
	color:rgb(255, 255, 255);
	font-weight: bold;
}
QPushButton:hover{
	background-color: rgb(0, 170, 255);
	cursor: pointer;
}</string>
     </property>
     <property name="text">
      <string>Quản lý dịch vụ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnQuanLyLichChieu">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>470</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font-size: 20px;
    border-radius: 10px;
	background-color: rgb(0, 85, 255);
	color:rgb(255, 255, 255);
	font-weight: bold;
}
QPushButton:hover{
	background-color: rgb(0, 170, 255);
	cursor: pointer;
}</string>
     </property>
     <property name="text">
      <string>Quản lý lịch chiếu</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnBaoCao">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>540</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font-size: 20px;
    border-radius: 10px;
	background-color: rgb(0, 85, 255);
	color:rgb(255, 255, 255);
	font-weight: bold;
}
QPushButton:hover{
	background-color: rgb(0, 170, 255);
	cursor: pointer;
}</string>
     </property>
     <property name="text">
      <string>Báo cáo doanh thu</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDangXuat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>610</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font-size: 20px;
    border-radius: 10px;
	background-color: rgb(255, 0, 0);
	color:rgb(255, 255, 255);
	font-weight: bold;
}
QPushButton:hover{
	background-color: rgb(255, 82, 82);
	cursor: pointer;
}</string>
     </property>
     <property name="text">
      <string>Đăng xuất</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>701</width>
      <height>701</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/istockphoto-1170311249-170667a.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
